src/nanoserver/1803/helix/amd64/Dockerfile

```diff
@@ -24,7 +24,16 @@ RUN python -m pip install --upgrade pip \
                              pyopenssl==19.0.0 \
                              requests==2.21.0 \
                              six==1.12.0 \
-                             virtualenv==16.5.0
+                             virtualenv==16.5.0 \
+    && pwsh -Command \
+    New-Item -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\' -Name 'dotnet.exe' -Force -ErrorAction SilentlyContinue ; \
```

Comment:
For readability, I would indent all of these Powershell commands so it's clear they're all a continuation of the call to `pwsh -Command`.  Same comment applies to the other file.

Reply:
Sure!